Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Management/Booking.hs

```diff
@@ -136,7 +140,7 @@ postBookingSyncMultiple merchantShortId opCity req = do
                     DRide.CANCELLED -> DBooking.CANCELLED
               let mbCancellationReason =
                     if bookingNewStatus == DBooking.CANCELLED && booking.status /= DBooking.CANCELLED
-                      then Just $ mkBookingCancellationReason merchant.id Common.syncBookingCode (Just ride.id) distanceUnit bookingId
+                      then Just $ mkBookingCancellationReason merchant.id (DMOC.id <$> mbMerchantOpCity) Common.syncBookingCode (Just ride.id) distanceUnit bookingId
```

Comment:
you can simplify here` (Just merchantOpCity.id)` instead of `(DMOC.id <$> mbMerchantOpCity)`